src/components/Footer/Footer.jsx

```diff
@@ -1,8 +1,86 @@
+                    <div className={styles.contact}>
+                        <div className={styles.buttondiv2}>
+                            <button className={styles.btn2}>
+                                <a href="">VISIT TECNOESIS</a>
+                            </button>
+                        </div>
+                        <p className={styles.pconnect}>CONNECT WITH US :</p>
+                        <div className={styles.social}>
+                            <div className={styles.imgdiv}><img src="https://res.cloudinary.com/dxafdfvui/image/upload/v1705083563/instapxl_sj0sgp.svg" alt="instagram" /></div>
+                            <div className={styles.imgdiv}><img src="https://res.cloudinary.com/dxafdfvui/image/upload/v1705083563/fbpxl_imhsav.svg" alt="facebook" /></div>
+                            <div className={styles.imgdiv}><img src="https://res.cloudinary.com/dxafdfvui/image/upload/v1705083935/twitterlogo_al9ufe.svg" alt="twitter" /></div>
+                        </div>
+                        <p className={styles.con}>CONTACT US</p>
+                        <p className={styles.gmail}>nitshack@gmail.com</p>
+                    </div>
+                    <div className={styles.logo}>
+                        <div className={styles.logoInner}>
+                            <img src="https://res.cloudinary.com/dxafdfvui/image/upload/v1705127726/logo_vfazzi.svg" alt="Logo" />
+                        </div>
+                    </div>
+                    <div className={styles.maps}>
+                        <p>How to Reach</p>
+                        <div className={styles.mapsimg}>
+                            {/* <div className={styles.map}>
+                                <h2 className={styles.mapH2}>Come Visit Us At Our Campus</h2>
+
+                                <div className={styles.googleMap}>
+                                    <GoogleMapReact
+                                        bootstrapURLKeys={{ key: '' }}
+                                        defaultCenter={location}
+                                        defaultZoom={zoomLevel}
+                                    >
+                                        <LocationPin
+                                            lat={location.lat}
+                                            lng={location.lng}
+                                            text={location.address}
+                                        />
+                                    </GoogleMapReact>
+                                </div>
+                            </div> */}
+                        </div>
+
+
+                    </div>
+                    <div className={styles.buttondiv}>
+                        <button className={styles.btn2}>
+                            <a href="">VISIT TECNOESIS</a>
+                        </button>
+                    </div>
+                </div>
+                <div className={styles.upper2}>
```

Comment:
![image](https://github.com/gdsc-nits-org/nits-hacks-6.0/assets/97835955/3d08e23d-f89e-429a-942a-b1c2113ccaaf)

try to make the changes exactly as given in the design